CHANGELOG.md

```diff
@@ -1,3 +1,154 @@
+# 0.2 - XXX, 2025 - "Natively Asynchronous Splicing"
+
+## API Updates
+
+ * Splicing is now supported. The current implementation is expected to be
```

Comment:
Curious why it says "expected". Isn't interop confirmed?

Reply:
We have not had time to do interop testing, no. We will likely do some before the release but probably not a *ton*.